cves/CVE-2017-7866.yml

```diff
@@ -64,7 +64,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A function in a file is expecting a stack. When the file gets the stack, it then
+  tries to write to it, but it writes expecting size that is 1 bigger than
+  the stack. This causes a stack overflow and can cause errors and the
+  application to crash.
```

Comment:
Add some elaboration in the description as to why the vulnerability had a security severity as high